packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -73,6 +71,7 @@ const MessageContentWithLiveAria = (props: MessageContentWithLiveAriaProps): JSX
 };
 
 const MessageContentAsRichTextHTML = (props: ChatMessageContentProps): JSX.Element => {
+  const htmlToReactParser = Parser();
```

Comment:
wrap this with useMemo so we don't have to create new object in every render
Or we could take one step forward - I guess we could just share the same parser and instructions instance between all components?
Is it possible to extract this part into a util function or something?